src/browser.ts

```diff
@@ -138,14 +138,63 @@ export class BrowserTab {
     }));
   }
 
-  private async saveAsMhtml(path: string) {
-    const cdp = await this._page.createCDPSession();
-    const { data } = await cdp.send('Page.captureSnapshot', { format: 'mhtml' });
-    await Bun.write(`${path}.mhtml`, data);
+  private async saveAsHtml(path: string) {
+    try {
+      await new Promise(resolve => setTimeout(resolve, 5000));
+
+      // Remove the cookie banner and fix the images
+      await this._page.evaluate(async () => {
+        document.querySelector('#onetrust-banner-sdk')?.remove();
+
+        // Fix the images
+        const images = document.querySelectorAll('img');
+        for (const img of Array.from(images)) {
+          const src = img.getAttribute('src');
+          if (src?.startsWith('/api/')) {
+            const response = await fetch('https://www.educative.io' + src);
```

Comment:
The code inside the callback `evaluate` is executed in the browser. So, this `fetch` is being executed by the browser not by the runtime (Bun). So, I believe we can remove `node-fetch` from the dependency.